sdk/eventhub/Azure.Messaging.EventHubs/src/Metadata/ConnectionStringProperties.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Messaging.EventHubs.Metadata
+{
+    /// <summary>
+    ///   The set of properties that comprise a connection string from the
+    ///   Azure portal.
+    /// </summary>
+    ///
+    public struct ConnectionStringProperties
```

Comment:
Does this have to be public?

Reply:
Due to the protected nature of the client methods, yes.

Reply:
With the internalization of connection string parsing, this no longer needs to be public and has been rescoped to `internal`